themes/bootstrap3/js/lightbox.js

```diff
@@ -131,21 +131,22 @@ VuFind.register('lightbox', function Lightbox() {
           VuFind.refreshPage();
           return;
         }
+        var testDiv = $('<div/>').html(content);
+        var errorMsgs = testDiv.find('.flash-message.alert-danger:not([data-lightbox-ignore])');
         // Place Hold error isolation
-        if (obj.url.match(/\/Record/) && (obj.url.match(/Hold\?/) || obj.url.match(/Request\?/))) {
-          var testDiv = $('<div/>').html(content);
-          var error = testDiv.find('.flash-message.alert-danger');
-          if (error.length && testDiv.find('.record').length) {
-            showAlert(error[0].innerHTML, 'danger');
+        if (obj.url.match(/\/(?!Record|Hold|Request)\?/)) {
+          if (errorMsgs.length && testDiv.find('.record').length) {
+            showAlert(errorMsgs[0].innerHTML, 'danger');
```

Comment:
Rather than only showing the first message, would it make sense to concatenate all of them, in case there's a series of errors?